content/docs/concepts/traffic-management/index.md

```diff
@@ -798,6 +799,32 @@ to:
 
 - Encode subsets in Server Name Indication (SNI) headers.
 
+### Service subsets
```

Comment:
Please do not move this section. The concept has a deliberate content structure. Each section starts by describing the use cases or features before moving to the implementation information. That's why "Load balancing 3 subsets" came before "Service subsets". We can discuss the merits of this approach and make modifications accordingly but that would need to happen on a separate PR after we reach agreement of what would work for all sections.